dev/linters/shellcheck.sh

```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/env bash
 
-set -o errexit
+set -o errexit -o nounset
+
+workspace=$(bazel info workspace)
```

Comment:
Nit: Quote?

Reply:
Don't need quote in variable declaration.